cmd/upload.go

```diff
@@ -285,7 +285,7 @@ var streamCmd = &cobra.Command{
 	},
 }
 
-func startChunkedUpload(cmd *cobra.Command, allocationObj *sdk.Allocation, localPath, thumbnailPath, remotePath string, encrypt bool, chunkSize int, attrs fileref.Attributes, statusBar sdk.StatusCallback, isUpdate bool) error {
+func startChunkedUpload(cmd *cobra.Command, allocationObj *sdk.Allocation, localPath, thumbnailPath, remotePath string, encrypt bool, chunkSize int, attrs fileref.Attributes, statusBar sdk.StatusCallback, isUpdate, isRepair bool) error {
```

Comment:
The arguments length is too long, please put them into a struct to make it clean and easier to read

Reply:
fixed